test/test-style.js

```diff
@@ -113,11 +114,9 @@ function jsonDiff(actual, expected) {
 
   for (var i = 0; i < actualLines.length; i++) {
     if (actualLines[i] !== expectedLines[i]) {
-      return [
-        '#' + (i + 1) + '\x1b[0m',
-        '    Actual:   ' + escapeInvisibles(actualLines[i]),
-        '    Expected: ' + escapeInvisibles(expectedLines[i]),
-      ].join('\n');
+      return chalk`#${i + 1}{reset
```

Comment:
I'm a fan of using the string templating when we can, but this is inconsistent with the rest of the files.  We should probably choose one and stick with it (and I'd say that the templating method would be better), but I think that this is a decision we can make later.

Reply:
This is because currently, only the top&#x2011;level `chalk` function can be&nbsp;a&nbsp;template function: https://github.com/chalk/chalk/issues/341